.github/workflows/docker-publish.yml

```
@@ -21,6 +25,37 @@ jobs:
         if: ${{ github.event.inputs.tag }}
         run: GITHUB_TAG=${{ github.event.inputs.tag }}; echo ::set-output name=tag::${GITHUB_TAG#grain-}
 
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1.6.1
+        with:
+          role-to-assume: arn:aws:iam::881972110175:role/github-actions-openid
+          role-session-name: cisession
+          aws-region: us-east-1
+
+      - name: Start AWS EC2 arm64 instance
+        id: instance
+        run: |
+          aws ec2 start-instances --instance-ids ${{ secrets.AWS_EC2_INSTANCE_ID }}
+          aws ec2 wait instance-running --instance-ids ${{ secrets.AWS_EC2_INSTANCE_ID }}
+          echo ::set-output name=ip::$(aws ec2 describe-instances --filters "Name=instance-id,Values=${{ secrets.AWS_EC2_INSTANCE_ID }}" --query 'Reservations[*].Instances[*].[PublicIpAddress]' --output text)
```

```suggestion
          aws ec2 start-instances --instance-ids ${{ secrets.AWS_EC2_INSTANCE_ID }}
          aws ec2 wait instance-running --instance-ids ${{ secrets.AWS_EC2_INSTANCE_ID }}
          echo ::set-output name=ip::$(aws ec2 describe-instances --filters "Name=instance-id,Values=${{ secrets.AWS_EC2_INSTANCE_ID }}" --query 'Reservations[*].Instances[*].[PublicIpAddress]' --output text)
```
We discussed not making this a secret since it doesn't **need** to be and might help with debugging.